lib/eapeak/parse.py

```diff
@@ -85,7 +94,7 @@ def mergeWirelessNetworks(source, destination):
 	for bssid in source.bssids:
 		destination.addBSSID(bssid)
 	
-	for mac, clientobj in source.clients.items():
+	for clientobj in source.clients.items():
```

Comment:
I think this is going to be changing the type, if only the `clientobj` is necessary then `.items()` should be changed to `.values()`.